src/Casdoor.Client/Abstractions/ICasdoorTokenClient.cs

```diff
@@ -1,10 +1,9 @@
-﻿using Casdoor.Client.Entity;
-using IdentityModel.Client;
+﻿using IdentityModel.Client;
```

Comment:
Please add Apache license headers for all .cs files, with "Casdoor authors" name.

Reply:
Ok